src/components/shared/CustomInput/index.js

```diff
@@ -0,0 +1,4 @@
+import styles from "./CustomInput.module.scss"
+
+export const CustomInput = ({ enterFunction, name }) => <input type="number" onKeyPress={enterFunction} name={name} className={styles.customInput} />
```

Comment:
rename the enterFunction to onChange and use onChange input handler instead of onKeyPress